core/src/main/java/feign/RequestTemplateFactoryResolver.java

```diff
@@ -47,7 +47,7 @@ public RequestTemplate.Factory resolve(Target<?> target, MethodMetadata md) {
     }
   }
 
-  private static class BuildTemplateByResolvingArgs implements RequestTemplate.Factory {
```

Comment:
Make those classes package private, so I don't need to copy them in `feign-vertx` project.